code/modules/mining/machine_vending.dm

```diff
@@ -55,6 +55,8 @@
 		new /datum/data/mining_equipment("Drone Ranged Upgrade",/obj/item/device/mine_bot_ugprade/cooldown,								600),
 		new /datum/data/mining_equipment("Drone AI Upgrade",	/obj/item/slimepotion/sentience/mining,									1000),
 		new /datum/data/mining_equipment("Jump Boots",			/obj/item/clothing/shoes/bhop,											2500),
+		new /datum/data/mining_equipment("Anti-Ash Walker injector",	/obj/item/weapon/storage/box/scalerot/pen,						2000),
```

Comment:
Remove the pretence, if this was really Anti-Ash Walker it'd be targeting their species and their species alone.
Especially since, and this is highly amusing, none of the output messages even work for Ash Walkers, so they'll know nothing until stage 5 (besides random damage and random stuns)